File: CPP/Day25.cpp
/*

Day 25:	Print the following pattern:
            * * * * *
             * * *
               *
*/


//Program
#include <iostream>
using namespace std;

int main() {

    int rows= 3;

    for(int i = rows; i >= 1; --i) {
        for(int space = 0; space < rows-i; ++space)
            cout << "  ";

        for(int j = i; j <= 2*i-1; ++j)
            cout << "* ";

        for(int j = 0; j < i-1; ++j)
            cout << "* ";

        cout << endl;
    }

    return 0;
}



/*

Output:

* * * * *
  * * *
    *

*/

File: Python/Day25.py
"""

Day 25:	Print the following pattern:
            * * * * *
             * * *
               *

"""

#Program
rows = 3
for i in range(rows, -1, -1):
    for space in range(0, rows-i):
        print("  ", end="")
    for j in range(i, 2*i):
        print("* ", end="")
    for j in range(0, i-1):
        print("* ", end="")
    print()
    
    


""" 

Output:

* * * * * 
  * * * 
    * 
    
"""
